tests/Doctrine/Tests/DBAL/Schema/MySqlSchemaManagerTest.php

```diff
@@ -37,6 +38,79 @@ public function testCompositeForeignKeys()
         $this->assertEquals(array('column_1', 'column_2', 'column_3'), array_map('strtolower', $fkeys[0]->getForeignColumns()));
     }
 
+    public function testPortableTableColumnDefinition()
+    {
+        $eventManager = new EventManager();
+        $driverMock = $this->createMock('Doctrine\DBAL\Driver');
+
+        $platforms = [
+            'mysql' => new \Doctrine\DBAL\Platforms\MySqlPlatform(),
+            'mariadb10.2' => new \Doctrine\DBAL\Platforms\MariaDb102Platform()
```

Comment:
It should be implemented as a data provider. Otherwise, if an assertion for MySQL fails, MariaDB won't even be tested.